### PR TITLE
update tests according to flowHistoryStatuses

### PR DESCRIPTION
**Pull request by andriidovhan**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistoryV2Spec.groovy [hunk 1]
```diff
@@ -198,6 +199,14 @@ class FlowHistoryV2Spec extends HealthCheckSpecification {
         then: "History is still available for the deleted flow"
         northbound.getFlowHistory(flow.flowId, specStartTime, timestampAfterUpdate).size() == 2
 
+        and: "Flow history statuses returns all flow statuses during for the whole life cycle"
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistoryV2Spec.groovy:202]:
```suggestion
        and: "Flow history statuses returns all flow statuses for the whole life cycle"
```

File: src-java/testing/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceV2Impl.java [hunk 2]
```diff
@@ -159,6 +160,43 @@ public FlowLoopResponse deleteFlowLoop(String flowId) {
...
+        return getFlowHistoryStatuses(flowId, timeFrom, timeTo, null);
+    }
+
+    @Override
+    public List<FlowHistoryStatusesResponse> getFlowHistoryStatuses(String flowId, Integer maxCount) {
+        return getFlowHistoryStatuses(flowId, null, null, maxCount);
+    }
+
+    @Override
+    public List<FlowHistoryStatusesResponse> getFlowHistoryStatuses(String flowId, Long timeFrom, Long timeTo,
```
Comment (rtretyak) [inline src-java/testing/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceV2Impl.java:179]: Why List?

**Review by rtretyak — CHANGES_REQUESTED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistoryV2Spec.groovy (hunk 1 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistoryV2Spec.groovy]: fixed

File: src-java/testing/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceV2Impl.java (hunk 2 above)
Comment (andriidovhan) [reply src-java/testing/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceV2Impl.java]: my bad, fixed

**Review by rtretyak — APPROVED** [review]

**Pull request merged**